### PR TITLE
[zh] sync configure-access-multiple-clusters configure-cgroup-driver resize-container-resources

### PR DESCRIPTION
**Pull request by asa3311**

### CODE REVIEW

File: content/zh-cn/docs/tasks/administer-cluster/kubeadm/configure-cgroup-driver.md [hunk 1]
```diff
@@ -66,12 +66,20 @@ kubeadm 支持在执行 `kubeadm init` 时，传递一个 `KubeletConfiguration`
...
+cgroup driver as an alpha feature.
+See [systemd cgroup driver](/docs/setup/production-environment/container-runtimes/#systemd-cgroup-driver)
+for more details.
 -->
 
 {{< note >}}
 在版本 1.22 及更高版本中，如果用户没有在 `KubeletConfiguration` 中设置 `cgroupDriver` 字段，
 `kubeadm` 会将它设置为默认值 `systemd`。
+
+在 Kubernetes v1.28 中，你可以以 Alpha 功能启用 cgroup 驱动程序的自动检测。
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/administer-cluster/kubeadm/configure-cgroup-driver.md:81]:
```suggestion
在 Kubernetes v1.28 中，你可以以 Alpha 功能启用 cgroup 驱动的自动检测。
```

File: content/zh-cn/docs/tasks/administer-cluster/kubeadm/configure-cgroup-driver.md [hunk 2]
```diff
@@ -66,12 +66,20 @@ kubeadm 支持在执行 `kubeadm init` 时，传递一个 `KubeletConfiguration`
...
+有关更多详情，请查看 [systemd cgroup 驱动程序](/zh-cn/docs/setup/production-environment/container-runtimes/#systemd-cgroup-driver)。
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/administer-cluster/kubeadm/configure-cgroup-driver.md:82]:
```suggestion
有关更多详情，请查看 [systemd cgroup 驱动](/zh-cn/docs/setup/production-environment/container-runtimes/#systemd-cgroup-driver)。
```

**Review by ydFu — APPROVED** [review]
/approve

**Pull request merged**